src/libs/runtime-utils.js

```diff
@@ -21,13 +21,33 @@ export const computeStyles = {
   warningView: { backgroundColor: colors.warning(0.1) }
 }
 
+export const pdTypes = {
+  standard: {
+    label: 'pd-standard',
+    displayName: 'Standard',
+    regionToPricesName: 'monthlyStandardDiskPrice'
+  },
+  ssd: {
+    label: 'pd-ssd',
+    displayName: 'Solid state drive (SSD)',
+    regionToPricesName: 'monthlySSDDiskPrice'
+  },
+  fromString: str => Utils.switchCase(str,
+    [pdTypes.standard.label, () => pdTypes.standard],
+    [pdTypes.ssd.label, () => pdTypes.ssd],
+    [Utils.DEFAULT, () => console.error('Should not be calling pdTypes. fromString for $(str), invalid disk type')]
+  )
+}
+export const mapDisksToPDTypes = disks => _.map(_.update('diskType', pdTypes.fromString), disks)
```

Comment:
```suggestion
export const mapToPdTypes = _.map(_.update('diskType', pdTypes.fromString))
```

since
1. we've been treating `Pd` as a word as far as camel-casing
2. we can take advantage of Lodash/FP's auto-curried nature and generalize the function for concision and flexibility. Its call sites shouldn't have to change other than the renaming.